src/redis/BetterSocialCronQueue.js

```diff
@@ -80,6 +80,10 @@ class BetterSocialCronQueue extends BetterSocialQueue {
       return refreshAllMaterializedViewProcess(job, done);
     }
 
+    if (flag === 'topicAutoMessage') {
+      return topicAutoMessageProcess(job, done);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional check for the 'topicAutoMessage' flag and subsequent call to `topicAutoMessageProcess(job, done)` is correct and follows the pattern established by the other cron jobs in this file. However, ensure that error handling is properly implemented within the `topicAutoMessageProcess` function itself, as any errors thrown could potentially halt the execution of subsequent jobs in the queue.
```diff
+    if (flag === 'topicAutoMessage') {
+      return topicAutoMessageProcess(job, done);
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->